include/nbl/asset/IImage.h

```diff
@@ -110,16 +112,7 @@ class IImage : public IDescriptor
 			ET_3D,
 			ET_COUNT
 		};
-		enum E_SAMPLE_COUNT_FLAGS : uint8_t
-		{
-			ESCF_1_BIT = 0x01,
-			ESCF_2_BIT = 0x02,
-			ESCF_4_BIT = 0x04,
-			ESCF_8_BIT = 0x08,
-			ESCF_16_BIT = 0x10,
-			ESCF_32_BIT = 0x20,
-			ESCF_64_BIT = 0x40
-		};
+		using E_SAMPLE_COUNT_FLAGS = nbl::hlsl::SampleCountFlags;
```

Comment:
this was a `uint8_t` to change it to a `uint16_t` would mess with a lot of things bytewise

You need to "alias" it differently, by defining `E_SAMPLE_COUNT_FLAGS` entries in terms of `SampleCountFlags` entries but with global conversion operators to and from each other